ui/plugins/auto_classification/panel.html

```diff
@@ -16,6 +16,7 @@
    ></th-autoclassify-toolbar>
 
 <div ng-switch="$ctrl.loadStatus">
+  <span ng-switch-when="job_pending">Job not complete, please wait</span>
```

Comment:
Kind of out of scope for this bug, but it might be nice to put a progress spinner in here + below, like this:

```html
<i aria-hidden="true" class="fa fa-spinner fa-pulse"></i>
```